packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -291,7 +328,78 @@ export class EcsOptimizedAmi implements ec2.IMachineImage {
     const ami = ssm.StringParameter.valueForStringParameter(scope, this.amiParameterName);
     return {
       imageId: ami,
-      osType: ec2.OperatingSystemType.LINUX
+      osType: this.windowsVersion ? ec2.OperatingSystemType.WINDOWS : ec2.OperatingSystemType.LINUX
+    };
+  }
+}
+
+/**
+ * Construct a Linux or Windows machine image from the latest ECS Optimized AMI published in SSM
+ */
+export class EcsOptimizedAmiStatic implements ec2.IMachineImage {
```

Comment:
Why is this called `Static`? If you are looking for a name that doesn't cause a breaking change, use `EcsOptimizedImage` or `EcsOptimizedMachineImage`